### ISSUE TITLE
1.3.0 build's bird isn't flocking

### ISSUE THREAD

**Issue by lcb931023** [after12am/boids.js#23]
Noticed a bug with the 1.3.0 build - the THREE.Bird's flock function isn't using the parameter to flock, but rather passing the entire `boids` object to its vehicle's flock function.

```
c.THREE.Bird.prototype.flock=function(a){this.behavior.flock(boids)
```
I tried changing `boids` to `a` in the minified file, and the function worked. Should be an easy fix 😄

**Comment by after12am** [comment]
Sorry late reply. I will check it today.

**Comment by after12am** [comment]
@lcb931023 Thank you for your issue! I fixed it 👍 628b2cf6405a1d2070c90fda7eb2fb111aa811e5

**Comment by lcb931023** [comment]
Awesome, thanks for the library!
